Godeps/_workspace/src/github.com/docker/libcompose/docker/service.go

```diff
@@ -59,15 +60,31 @@ func (s *Service) collectContainers() ([]*Container, error) {
 	for _, container := range containers {
 		name := container.Labels[NAME.Str()]
 		c := NewContainer(client, name, s)
-		if outOfSync, err := c.OutOfSync(imageName); err != nil {
-			return nil, err
-		} else if outOfSync && s.context.Rebuild && s.Config().Labels.MapParts()[REBUILD.Str()] != "false" {
-			logrus.Infof("Rebuilding %s", name)
-			if _, err := c.Rebuild(imageName); err != nil {
+		if s.context.Rebuild {
```

Comment:
@ibuildthecloud We're even considering rebuilding only if `s.context.Rebuild` is true, correct?